src/legacy/server/config/schema.js

```diff
@@ -205,11 +205,11 @@ export default () =>
       opensearchManifestServiceUrl: Joi.string().default(
         'https://maps.search-services.aws.a2z.com/v4/ap-southeast-1/manifest'
       ),
-      emsFileApiUrl: Joi.string().default('https://vector.maps.elastic.co'),
-      emsTileApiUrl: Joi.string().default('https://tiles.maps.elastic.co'),
-      emsLandingPageUrl: Joi.string().default('https://maps.elastic.co/v7.9'),
+      emsFileApiUrl: Joi.string().default('https://vector.maps.opensearch.org'),
+      emsTileApiUrl: Joi.string().default('https://tiles.maps.opensearch.org'),
+      emsLandingPageUrl: Joi.string().default('https://maps.opensearch.org/v7.10'),
```

Comment:
Are these dead links? Does it affect the rendering in any way?

Reply:
> Are these dead links?
Yes but it removes the elastic.co stuff

> Does it affect the rendering in any way?
It uses `opensearchManifestServiceUrl` to actual get the map values. To be honest, there can be additional work that cleans up unneeded stuff.

Reply:
I just updated those values to real links. Better to be unused and real rather than used and fake.